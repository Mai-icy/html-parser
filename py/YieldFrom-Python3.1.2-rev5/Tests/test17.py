#
#   Test attempting to send to non-generator
#

def g():
	try:
		print("starting g")
		yield from range(3)
		print("g should not be here")
	finally:
		print("finishing g")

gi = g()
next(gi)
for x in range(3):
	y = gi.send(42)
	print("Should not have yielded:", y)
