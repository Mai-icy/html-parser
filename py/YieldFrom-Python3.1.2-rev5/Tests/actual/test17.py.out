starting g
finishing g
Traceback (most recent call last):
  File "test17.py", line 16, in <module>
    y = gi.send(42)
  File "test17.py", line 8, in g
    yield from range(3)
AttributeError: send
